setup.sh

```diff
@@ -2,21 +2,20 @@
 version=$1
 run_bootstrap=$2
 
-echo Installing Melos $version
-
-if [ -x "$(command -v flutter)" ]
-then
-  flutter pub global activate melos $version
-elif [ -x "$(command -v dart)" ]
-then
-  dart pub global activate melos $version
+if [ -x "$(command -v flutter)" ]; then
+  exec=flutter
+elif [ -x "$(command -v dart)" ]; then
+  exec=dart
 else
   echo 'Error: Either dart or flutter needs to be on the PATH.' >&2
   exit 1
 fi
 
-if [ "$run_bootstrap" != "false" ]
-then
-  echo Running melos bootstrap
-  melos bootstrap
+echo Installing Melos $version via $exec
+
+$exec pub global activate melos $version
```

Comment:
This should always be dart, doesn't work properly with flutter activate for some reason

Reply:
really? it was broken before then?

Reply:
We might not have run into the thing where it doesn't work, or it has been fixed, I thought that those two commands would be equivalent, but the `flutter` one used to cause issues when activating melos before. On their site it also says `dart` in the instructions. Maybe you could give some insight @salakhar?

Reply:
I've not got much thoughts on this sorry, I've always just done `dart pub global activate` so may be why I've never seen issues

Reply:
Thanks for the reply! Maybe it's not even an issue anymore.